public/assets/application-1e098f4116ba1464c4096a4f9c7e8646.js

```diff
@@ -0,0 +1 @@
+(function(){jQuery(function(){return $("a[rel~=popover], .has-popover").popover(),$("a[rel~=tooltip], .has-tooltip").tooltip()})}).call(this);var hideModal=function(){$("#submit-button").click(function(){$(".modal-content").remove()})};
```

Comment:
Line is too long.<br>Missing semicolon.<br>'hideModal' is defined but never used.